packages/eas-cli/src/build/ios/version.ts

```diff
@@ -97,6 +99,15 @@ export async function bumpVersionInAppJsonAsync({
   }
 }
 
+function validateShortVersion(shortVersion: string | undefined): string | undefined {
```

Comment:
I would expect that function called validate would not return anything, and if want a function that "validates", but can for example modify  the value then we usually call function like that sanitize.